assets/styles/utils.scss

```diff
@@ -14,3 +14,17 @@ body {
 .text-container p {
   line-height: 1.3;
 }
+
+.sr-only {
+  border: 0 !important;
+  clip: rect(1px, 1px, 1px, 1px) !important;
+  -webkit-clip-path: inset(50%) !important;
+  clip-path: inset(50%) !important;
+  height: 1px !important;
+  margin: -1px !important;
+  overflow: hidden !important;
+  padding: 0 !important;
+  position: absolute !important;
+  width: 1px !important;
+  white-space: nowrap !important;
+}
```

Comment:
what is this used for?

Reply:
It makes text only readable for screen readers, which is useful in notices to denote information type provided them (like ‘Warning:’), e.g.: [`ca7a5c4` (#738)](https://github.com/modrinth/knossos/pull/738/commits/ca7a5c4ad105e5cabaa34f4db4dd3d397e5f4738#diff-17552590ddc5cadd71d99ab273e508237ae8d5426c32eca78a313b1186b5e9e7R20-R26)

You used `aria-label` for that, but I remember reading that this is incorrect usage for that, and since it's very easy fix, I just made it part of this PR.

Additional unspoken benefit is ability for browsers to modify this when they're automatically translating the page, since they won't touch attributes, but again, that's only useful for screen readers and text-only navigators.